test/phpstan-php-version.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types = 1);
+
+return [
+    'parameters' => [
+        'phpVersion' => PHP_VERSION_ID,
```

Comment:
We could also drop `config.platform.php` from `composwr.json`: the reason it was added was to prevent invalid @dependabot upgrades, but we can get rid of it, if it has such an influence on other tools too...

Reply:
Yeah, `"php": "~8.0.12 || ~8.1.0"` might be sufficient to keep valid dependencies.

Reply:
Interesting, the changed dropped `composer/package-versions-deprecated` and installed `ocramius/package-versions`, not sure why is that :) https://github.com/Roave/BetterReflection/pull/945/commits/8812e7bce75c6fdffa83b870299335669866a2a1

Reply:
Massive improvement, I'd say 🤣

Reply:
Funny.

I've already removed `composer/package-versions-deprecated` in https://github.com/Roave/BetterReflection/commit/bae419ad6fff85c5767daf2839862bc27985bc3d

But it returned back in: https://github.com/Roave/BetterReflection/commit/24320f80c59846bacf7fc536276dab088cbe1eaf